src/Microsoft.DotNet.Compiler.Common/AssemblyInfoFileGenerator.cs

```diff
@@ -68,9 +68,15 @@ public static string GenerateFSharp(AssemblyInfoOptions metadata)
                 [typeof(AssemblyVersionAttribute)] = EscapeCharacters(metadata.AssemblyVersion?.ToString()),
                 [typeof(AssemblyInformationalVersionAttribute)] = EscapeCharacters(metadata.InformationalVersion),
                 [typeof(AssemblyCultureAttribute)] = EscapeCharacters(metadata.Culture),
-                [typeof(NeutralResourcesLanguageAttribute)] = EscapeCharacters(metadata.NeutralLanguage),
-                [typeof(TargetFrameworkAttribute)] = EscapeCharacters(metadata.TargetFramework)
+                [typeof(NeutralResourcesLanguageAttribute)] = EscapeCharacters(metadata.NeutralLanguage)
             };
+            
+            // only .net 4.0+ compatible
+            if (metadata.TargetFrameworkVersion == null || metadata.TargetFrameworkVersion >= new Version(4, 0))
```

Comment:
You're right. I just found that there is a NuGetFramework.Parse method, so I'll use that and dump the TargetFrameworkVersion again.

Concerning tests. I'm happy to add one, but I'll need some guidance. Should I add a new project in TestAssets\TestProjects which targets multiple frameworks (incl. net20), or is there already something to reuse? Then add a new test class in 'dotnet-build.Tests' like 'IncrementalTestsMultiTargetFrameworks'?